packages/compass/src/main/auto-update-manager.ts

```diff
@@ -579,9 +579,44 @@ class CompassAutoUpdateManager {
       ...options,
     };
 
+    // TODO: does it make sense to still support this? Should the menu item just
+    // be hidden?
+    compassApp.on('check-for-updates', () => {
+      this.setState(AutoUpdateManagerState.ManualCheck);
+    });
+
+    const supported = supportsAutoupdater();
     const enabled = !!preferences.getPreferences().autoUpdates;
 
+    log.info(
+      mongoLogId(1001000133),
+      'AutoUpdateManager',
+      'Setting up updateManager',
+      { ...this.autoUpdateOptions, supported, enabled }
+    );
+
+    if (!supported) {
+      log.info(
+        mongoLogId(1_001_000_247),
+        'AutoUpdateManager',
+        `autoUpdate not supported'}`
+      );
```

Comment:
The previous `log` message covers this, no?

Reply:
yeah this is old. I'll remove it.